controller/pkg/aclprovider/iptablesprovider.go

```diff
@@ -66,6 +66,7 @@ const (
 func NewGoIPTablesProviderV4(batchTables []string) (IptablesProvider, error) {
 	ipt, err := iptables.New()
 	if err != nil {
+		zap.L().Error("Failed to create new go-iptables provider ", zap.Error(err))
```

Comment:
This error is not required. If you look into the call chain, the caller prints the error. We will have duplicate error messages.

Reply:
Last night I also saw that the caller prints the error in our call chain, but I actually put this in because when unit testing with the **testing.T** framework it fails silently without saying why because the test code calls this function **NewGoIPTablesProviderV4** directly and does not print the err. I could also print the error in the test calls too instead of here. 

I'll remove it from here.